cloud/aws/services/ec2/instances_test.go

```diff
@@ -208,7 +208,12 @@ metadata:
 spec:
   versions:
     kubelet: v1.11.2
-    controlPlane: v1.11.2`),
+    controlPlane: v1.11.2
+  providerConfig:
+  	value:
```

Comment:
There is a tab here, should be two spaces, but also this YAML isn't used. the expect is getting values from the call, not from the raw extension. Not terribly important, but probably good to be consistent with other tests